cypress/features/regression/test/badge.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Badge component
+  I want to change Badge component properties
+
+  Background: Open Badge component page
+    Given I open basic Test "Badge" component page
+
+  @positive
+  Scenario: Badge component rendered properly
+    # commented because of BDD default scenario Given - When - Then
+    # When I open basic Test "Badge" component page
+    Then Badge component rendered properly
+
+  @positive
+  Scenario Outline: Set Badge component to <counter>
+    When I set counter to "<counter>"
+    Then Badge component counter is set to <counter>
+    Examples:
```

Comment:
I don't think we need 4 examples. Remove `1` or `5`

Reply:
updated